tools/buildsteps/windows/buildffmpeg.sh

```diff
@@ -98,8 +98,8 @@ export CFLAGS=""
 export CXXFLAGS=""
 export LDFLAGS=""
 
-extra_cflags="-I$LOCALDESTDIR/include -I/depends/$TRIPLET/include -DWIN32_LEAN_AND_MEAN"
-extra_ldflags="-LIBPATH:\"$LOCALDESTDIR/lib\" -LIBPATH:\"$MINGW_PREFIX/lib\" -LIBPATH:\"/depends/$TRIPLET/lib\""
+extra_cflags="-I$LOCALDESTDIR/include -I/depends2/$TRIPLET/include -DWIN32_LEAN_AND_MEAN"
```

Comment:
Why `depends2`?

Reply:
It's a trick in MSYS to maintain backwards compatibility with Matrix since the MSYS2 installation is shared.

In the `fstab` file the links with external/internal paths (Windows/MSYS2) are defined:

```
T:\KODI\kodi\project\BuildDependencies\build\            /build
T:\KODI\kodi\project\BuildDependencies\downloads\        /downloads
T:\KODI\kodi\project\BuildDependencies\locals\win32\     /local32
T:\KODI\kodi\project\BuildDependencies\locals\x64\       /local64
T:\KODI\kodi\project\BuildDependencies\msys64\mingw32\  /mingw32
T:\KODI\kodi\project\BuildDependencies\msys64\mingw64\  /mingw64
T:\KODI\kodi\project\BuildDependencies\downloads2\       /var/cache/pacman/pkg
T:\KODI\kodi\project\BuildDependencies\win32\            /depends/win32
T:\KODI\kodi\project\BuildDependencies\x64\              /depends/x64
T:\KODI\kodi\project\BuildDependencies\win10-arm\        /depends/win10-arm
T:\KODI\kodi\project\BuildDependencies\win10-win32\      /depends/win10-win32
T:\KODI\kodi\project\BuildDependencies\win10-x64\        /depends/win10-x64
T:\KODI\kodi\project\BuildDependencies\..\..\            /xbmc
T:\KODI\kodi\project\BuildDependencies\build-20\win32\           /depends2/win32
T:\KODI\kodi\project\BuildDependencies\build-20\x64\             /depends2/x64
T:\KODI\kodi\project\BuildDependencies\build-20\win10-arm\       /depends2/win10-arm
T:\KODI\kodi\project\BuildDependencies\build-20\win10-win32\     /depends2/win10-win32
T:\KODI\kodi\project\BuildDependencies\build-20\win10-x64\       /depends2/win10-x64
```
i.e 

`/depends2/x64`  --------------> `T:\KODI\kodi\project\BuildDependencies\build-20\x64\ `

Matrix still needs access to old style path:
`/depends/x64 `---------------->`T:\KODI\kodi\project\BuildDependencies\x64\ `

If backported this PR to Matrix (not planed for now) then:
`/depends2/x64`  --------------> `T:\KODI\kodi\project\BuildDependencies\build-19\x64\ `

Reply:
Duplicated routes may be removed in the future when the "old" system is no longer in use.